### DISCUSSION TITLE
Meeting address correct in the plugin, incorrect in the meeting guide app

### DISCUSSION THREAD

**Discussion by jackieoh1** [code4recovery/12-step-meeting-list#827] [Q&A] [answered]
Greetings - 

We created a meeting recently, the "What a Concept Group, for Northern Michigan Central Office (nmcentraloffice.org), at our office which address is 3291 Racquet Club Drive, Suite B, Traverse City, MI 49684.  It shows up correctly in the 12 step meeting list, but in the Meeting Guide app, it is showing "3287 Racquet Club Drive."  

I happen to know that "3287" is the address that Google interprets as the location, unfortunately.  Is this a new issue or a known issue?  We have had other meetings at the Central Office and I don't recall having the issue before.  

Should the Meeting Guide app take the geolocation and infer the address, or take the address string from the stated address and place the marker from the geolocation separately? (Seems like the latter).  

Thanks -  any guidance will be helpful!

Jodie/NMCO

**Comment by tim-rohrer** [comment, marked as answer]
Hi Jodie--

The short answer is that you'll probably need to contact appsupport@aa.org to fix this.

When I entered 3291 Racquet Club Drive, Suite B, Traverse City, MI 49684, at maps.google.com, "3291" was returned, which probably explains why things are correct on your website (unless you've also overridden the address in your local `functions.php`.

However, Meeting Guide also geocodes addresses a second time, and while it is rare, we occasionally see a different result. 

If you contact GSO/Meeting Guide, they should be able to override or otherwise help get this showing correctly in the app.

**Reply by jackieoh1** [reply]
Thanks Tim, I'll pursue it with them to get an override.
